core/jazz_t-vault/swagger/swagger.json

```diff
@@ -0,0 +1,3589 @@
+                    }
+                },
+                "responses": {
+                    "200": {
+                        "description": "Default response for CORS method",
+                        "headers": {
+                            "Access-Control-Allow-Headers": {
+                                "type": "string"
+                            },
+                            "Access-Control-Allow-Methods": {
+                                "type": "string"
+                            },
+                            "Access-Control-Allow-Origin": {
+                                "type": "string"
+                            }
+                        }
+                    },
+                    "400": {
+                        "description": "Bad Request",
+                        "schema": {
+                            "$ref": "#/definitions/400ErrorResponse"
+                        }
+                    },
+                    "401": {
+                        "description": "Unauthorized",
+                        "schema": {
+                            "$ref": "#/definitions/401ErrorResponse"
+                        }
+                    },
+                    "403": {
+                        "description": "Forbidden",
+                        "schema": {
+                            "$ref": "#/definitions/403ErrorResponse"
+                        }
+                    },
+                    "404": {
+                        "description": "Not Found",
+                        "schema": {
+                            "$ref": "#/definitions/404ErrorResponse"
+                        }
+                    },
+                    "500": {
+                        "description": "Internal Server Error",
+                        "schema": {
+                            "$ref": "#/definitions/500ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "post": {
+                "description": "create a safe in t-vault",
+                "parameters": [
+                    {
+                        "name": "param",
+                        "in": "body",
+                        "description": "post body parameter.",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/CreateSafeObj"
+                        }
+                    },
+                    {
+                      "name": "Jazz-Service-ID",
+                      "in": "header",
+                      "required": true,
+                      "type": "string"
+                    }
+                ],
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "security": [
+                    {
+                        "{envPrefix}-{domain}-cognito-authorizer": []
```

Comment:
Only `/safes/{safeName}/role` API should use this authorizer. All the remaining APIs - creation/deletion of safes/users, adding/removing users to the safe is done by admin user (hence uses `{envPrefix}-{domain}-cognito-admin-authorizer`)

Reply:
Done